printdb.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sqlite3
+
+from DAO import _Employees, _Suppliers, _Products, _CoffeeStands, _Activities
+from DTO import *
+
+
+def print_employees_report(cursor):
+    print("Employees report")
+    cursor.execute("""SELECT emp.name, emp.salary, cs.location, sum(prod.price * act.quantity)
+    FROM ((Employees AS emp
+    INNER JOIN Coffee_stands AS cs 
+    ON emp.coffee_stand = cs.id)
+    LEFT OUTER JOIN (Activities AS act
+    INNER JOIN Products AS prod ON act.product_id = prod.id)
+    ON act.activator_id = emp.id)
+    GROUP BY emp.id
+    """)
+
+    all_data = cursor.fetchall()
+    for one in all_data:
+        print(*one)
+
+
+# , sum(prod.price * act.quantity)
+# LEFT OUTER JOIN (Activities AS act
+#     INNER JOIN Products as prod on act.product_id = prod.id)
+#     ON emp.id = act.activator_id)
+#   ON emp.id = act.activator_id
+#   ORDER BY emp.name ASC
```

Comment:
Commented out code should probably be removed

Reply:
removed before even seeing this comment :)